treccovid_semantic_search/workload.json

```diff
@@ -0,0 +1,32 @@
+{% import "benchmark.helpers" as benchmark with context %}
+
+{
+  "version": 2,
+    "description": "Trec-Covid is a dataset collection of documents about COVID-19 information.",
+  "indices": [
+    {
+      "name": "treccovid",
+      "body": "index.json"
+    }
+  ],
+  "corpora": [
+    {
+      "name": "treccovid",
+      "base-url": "https://vesa-oswl.s3.us-west-2.amazonaws.com/treccovid",
```

Comment:
@vpehkone This can now be updated to `https://dbyiw3u3rf9yr.cloudfront.net/corpora/treccovid`. Please confirm if document count is also correct. Is `129192` from the uncompressed version of documents.json? 
```
$ wc -l documents.json.bz2
253809 documents.json.bz2
```

Reply:
Updated the new URL for documents and queries. Yes, the document count is right.